src/components/__tests__/OnboardingStepper.cy.js

```diff
@@ -0,0 +1,101 @@
+import { colors } from 'quasar';
+
+import OnboardingStepper from 'components/onboarding/OnboardingStepper.vue';
+import { i18n } from '../../boot/i18n';
+
+const { getPaletteColor } = colors;
+const black = getPaletteColor('black');
+const grey10 = getPaletteColor('grey-10');
+
+describe('<OnboardingStepper>', () => {
+  it('has translation for all strings', () => {
+    cy.testLanguageStringsInContext(['skip'], 'navigation', i18n);
+    cy.testLanguageStringsInContext(
+      [
+        'buttonInviteFriends',
+        'descriptionStep1',
+        'descriptionInvite',
+        'labelInviteEmailAddresses',
+        'labelLanguage',
+        'messageRequiredAddresses',
+        'textMessage',
+        'titleMessage',
+        'titleStep1',
+        'titleStep2',
+      ],
+      'onboarding',
+      i18n,
+    );
+  });
+
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.mount(OnboardingStepper, {
+        props: {},
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders component', () => {
+      // Step 1
+      cy.dataCy('onboarding-stepper').should('be.visible');
+      // title
+      cy.dataCy('step1-title')
+        .should('be.visible')
+        .and('have.css', 'font-size', '24px')
+        .and('have.css', 'font-weight', '700')
+        .and('have.css', 'margin-top', '0px')
+        .and('have.css', 'margin-bottom', '0px')
+        .and('have.color', black)
+        .and('contain', i18n.global.t('onboarding.titleStep1'));
+      // description
+      cy.dataCy('step1-description')
+        .should('be.visible')
+        .and('have.css', 'margin-top', '24px')
+        .and('have.css', 'font-size', '14px')
+        .and('have.css', 'font-weight', '500')
+        .and('have.color', grey10)
+        .and('contain', i18n.global.t('onboarding.descriptionStep1'));
+      // video
+      cy.dataCy('step1-video').should('be.visible');
```

Comment:
1. Yo can add test if video URL exists., please.

Use the same code for testing URL (and solving rate limit error), please.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/6456d667c1302d7d25928b0c2590986fc5682a43/src/components/__tests__/FooterBar.cy.js#L98-L107

3. You can add test for testing video playing https://glebbahmutov.com/blog/test-video-play/, please.

Reply:
@tmszi  added tests for the URL and the `src` attribute on the `iframe`.

As for point no. 3, Quasar's implementation of YouTube video embed is done via`iframe` rather than `video` element.
Since it is relatively complicated to [test iframes in Cypress](https://www.lambdatest.com/blog/how-to-handle-iframes-in-cypress/) I would rely on the existing infrastructure for the "play" and "pause" actions.

Reply:
Yes you are right. Ok I agree.